tools/hls-verifier/src/Utilities.cpp

```diff
@@ -46,6 +46,11 @@ bool FloatCompare::compare(const string &token1, const string &token2) const {
   stringstream is2(token2);
   is1 >> hex >> i1;
   is2 >> hex >> i2;
+
+  if (isnan(*((float *)&i1)) && isnan(*((float *)&i2))) {
+    return true;
+  }
```

Comment:
We write a simple if as:
```
if (isnan(*((float *)&i1)) && isnan(*((float *)&i2)))
  return true;
```

Reply:
Are you sure this is better, considering the rest of the file uses brackets regardless if there is one statement inside or more? E. g.

```if (filepath[i] == '/') {
      return filepath.substr(0, i);
    }```

Reply:
Yeah it would be great if you could also change the others (guideline: https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements)